integration/test/basic/fixtures/tb-chart/templates/test/test-runner.yaml

```diff
@@ -1,41 +0,0 @@
-apiVersion: v1
```

Comment:
This `tb-chart` is leftover from previous PR. In this PR I created another test-chart which is basically as other one, but w/o tests. So the diff looks like that.